packages/web/src/scss/components/Drawer/_DrawerPanel.scss

```
@@ -0,0 +1,43 @@
+@use 'sass:map';
+@use 'theme';
+
+.DrawerPanel {
+    --drawer-translate-x: 0%;
+
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    width: theme.$drawer-panel-width;
+    height: theme.$drawer-panel-height;
+    max-height: theme.$drawer-panel-max-height;
+    color: theme.$drawer-panel-text-color;
+    background-color: theme.$drawer-panel-background-color;
+    box-shadow: theme.$drawer-panel-shadow;
+    transform: translateX(var(--drawer-translate-x));
+
+    @media (prefers-reduced-motion: no-preference) {
+        transition-property: transform;
+        transition-duration: theme.$drawer-transition-duration;
+        transition-timing-function: theme.$drawer-transition-timing;
+    }
+}
+
+.Drawer--left .DrawerPanel {
```

Can we be more specific?
```suggestion
.Drawer--left > .DrawerPanel {
```
this applies to all of them